docs/source/configuration/open_edx.rst

```diff
@@ -150,6 +150,8 @@ In Open edX (derived from instructions `here <https://edx.readthedocs.io/project
     * Skip registration form
     * Sync learner profile data
     * Enable SSO id verification
+    * Enabled
```

Comment:
I think moving this to the top of this list will make it synced with how the `ProviderConfig` form shows it. 
<img width="1118" alt="Screenshot 2022-07-19 at 7 15 59 PM" src="https://user-images.githubusercontent.com/34372316/179772547-5007b924-3938-4d6d-9460-d64a13744e18.png">